_documentation/user/flags/colorblindness.md

```diff
@@ -5,4 +5,18 @@ type: Bad
 summary: Can't tell team colors. Don't shoot teammates!
 ---
 
-...
+The Colorblindness flag causes a player to see all players as rogue players and identify no longer identifies players by name for them.
+
+## Tactics
+
+Colorblindness causes teammates and enemies to look exactly alike. The risk of shooting a teammate increases dramatically and enemies can easily blend in with teammates.
+
+Countering the effects of Colorblindness requires good communications with teammates to know their positions and in order to locate enemies quickly.
+The main difficulties are that positions may quickly become outdated and also stealth may increase the risk of mistakenly shooting a player.
+Another recommendation is to view the radar frequently, especially if grabbing a flag, as it allows one to have a rough idea of the general location and position of teammates and enemies.
+While the main counter to Colorblindness is communications, this is not always possible, and thus Colorblindness may be one of the worst bad flags to have.
+
+## Server Variables
+The variables which affect the Colorblindness flag are: **None**
```

Comment:
If there are none, then there's no need for this section

Reply:
Noted.